common/org.testeditor.dsl.common.testing/src/org/testeditor/dsl/common/ide/util/ProjectContentGeneratorTest.xtend

```diff
@@ -0,0 +1,129 @@
+/*******************************************************************************
+ * Copyright (c) 2012 - 2016 Signal Iduna Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * 
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ * itemis AG
+ *******************************************************************************/
+package org.testeditor.dsl.common.ide.util
+
+import org.junit.Test
+import static org.junit.Assert.*
+import java.util.HashSet
+import javax.xml.parsers.DocumentBuilderFactory
+import java.io.ByteArrayInputStream
+import javax.xml.xpath.XPathFactory
+
+class ProjectContentGeneratorTest {
+
+	@Test
+	def testGetAvailableSelections() {
```

Comment:
This should be split up in two test cases as two methods are tested. I would extract the `val generator = new ProjectContentGenerator()` and let a just be a field of this class.